pkg/workers/notifications/notification.go

```diff
@@ -0,0 +1,211 @@
+package notifications
+
+import (
+	"crypto/ecdsa"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/cozy/cozy-stack/pkg/config"
+	"github.com/cozy/cozy-stack/pkg/jobs"
+
+	multierror "github.com/hashicorp/go-multierror"
+
+	fcm "github.com/appleboy/go-fcm"
+	apns "github.com/sideshow/apns2"
+	apns_cert "github.com/sideshow/apns2/certificate"
+	"github.com/sideshow/apns2/payload"
+	apns_token "github.com/sideshow/apns2/token"
+)
+
+const (
+	// Android platform using Firebase Cloud Messaging (FCM)
+	Android = "android"
+	// IOS platform using APNS/2
+	IOS = "ios"
+)
+
+var (
+	fcmClient *fcm.Client
+	iosClient *apns.Client
+)
+
+func init() {
+	jobs.AddWorker("notification", &jobs.WorkerConfig{
```

Comment:
Can you add documentation about this worker in https://github.com/cozy/cozy-stack/blob/master/docs/workers.md?

Reply:
Sure, will do.

FYI I'm still not really confident in the message type that is given to this worker. I built It as a kind of intersection between what I found for FCM (Android) and APNS (iOS) notifications, but it may not cover all use-cases, or worse, not contain enough information.